Main.hs

```diff
@@ -105,10 +105,7 @@ main = do
     dieWithUsage
 
   when verbose $
-    printf "GHC iserv starting (in: %d; out: %d)\n"
-      (fromIntegral rfd2 :: Int) (fromIntegral wfd1 :: Int)
-  inh  <- getGhcHandle rfd2
```

Comment:
`getGhcHandle` on windows is [`HANDLE -> IO Handle`](https://gitlab.haskell.org/ghc/ghc/-/blob/c06b534bbd6dbb570b760f2e82b3e375d3956bcf/libraries/ghci/GHCi/Utils.hsc#L31-32) where [`HANDLE = Ptr ()`](https://gitlab.haskell.org/ghc/ghc/-/blob/c06b534bbd6dbb570b760f2e82b3e375d3956bcf/libraries/ghc-internal/src/GHC/Internal/Windows.hs#L107) rather than [`CInt -> IO Handle`](https://gitlab.haskell.org/ghc/ghc/-/blob/c06b534bbd6dbb570b760f2e82b3e375d3956bcf/libraries/ghci/GHCi/Utils.hsc#L52) on unix

[readGhcHandle](https://gitlab.haskell.org/ghc/ghc/-/blob/c06b534bbd6dbb570b760f2e82b3e375d3956bcf/libraries/ghci/GHCi/Utils.hsc#L57) handles both cases